src/lib/mina/local-blockchain.ts

```diff
@@ -39,7 +40,23 @@ import {
   verifyAccountUpdate,
 } from './transaction-validation.js';
 
-export { LocalBlockchain };
+export { LocalBlockchain, TestAccount };
+
+class TestAccount extends PublicKey {
```

Comment:
I think this should be called `TestPublicKey`, not `TestAccount`, because `Account` suggests much more than just a keypair and we already have an `Account` type represents an onchain Mina account with many account fields.

Reply:
Or  `TestAddress`